guide/src/async_python_from_rust.md

```diff
@@ -0,0 +1,186 @@
+# Calling async Python functions in Rust code
+
+In the previous chapter, we learnt how to call Python functions in Rust code. Now, we will look at two ways to call Python async functions from Rust code.
+
+## Async Functions
+
+Before we start, let us take a look at the following snippet.
+
+```python
+async def fx():
+    return "hello, world"
+```
+
+Above, we have defined an async function. How we send it to Rust code will make a difference.
```

Comment:
> How we send it to Rust code will make a difference.

Can you try extend this sentence? It's not very clear what you mean here. The reader might wonder why does it make a difference? How can they send it to Rust code?